yal/utils.py

```diff
@@ -64,3 +66,12 @@ def normalise_phonenumber(phonenumber):
         return phonenumbers.format_number(pn, phonenumbers.PhoneNumberFormat.E164)
     except (phonenumbers.phonenumberutil.NumberParseException, AssertionError):
         raise ValueError("Invalid phone number")
+
+
+def replace_persona_fields(text, metadata={}):
+    for key in PERSONA_FIELDS:
+        if key in metadata:
+            text = text.replace(f"[{key}]", metadata[key])
```

Comment:
I'm guessing it's too late to rather use something like https://docs.python.org/3.8/library/string.html#string.Template? Since it uses a different placeholder format.

Reply:
yeah, unfortunately it is a bit too late. Codie, Erik and I discussed it and chose this format since it's applicable across platforms (python framework vs rapidpro) and doesn't clash with the Turn template. But we didn't consider this option. We also didn't take into account the clash with Markdown's formatting.
but unfortunately the content has already been formatted this way. I think we can reopen this format conversation after the mvp